src/components/pokemon-selected/showPokemonCompare.js

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import "./pokemonCompare.module.css";
+import DobleCharts from "../chartsCompareFolder/chartsCompare";
+import ScrollLock from 'react-scrolllock';
+
+const ShowPokemonCompare = ({
+  pokemonInPokeball,
+  ViewState,
+  cleanPokemonArray,
+}) => (
+  <div
+    className={ViewState ? "compare-container-chart" : "hidden"}
+    onclick={cleanPokemonArray}
+  >
+    <ScrollLock isActive={ViewState}> 
+    <div className="pokecard-compare-container">
+      <div className="versus-names">
+        {String(pokemonInPokeball[0].name).toUpperCase()} VS.{" "}
+        {String(pokemonInPokeball[1].name).toUpperCase()}
+        <button className="compare-close-button" onClick={cleanPokemonArray}>
+          X
+        </button>
+      </div>
+      <div className="bodyChart">
+        <div className="images-box">
+          <img
+            src={`${pokemonInPokeball[0].image}`}
+            className="image-compare"
+            alt="pokemons"
+          />
+          <img
+            src={`${pokemonInPokeball[1].image}`}
+            className="image-compare"
+            alt="pokemons"
+          />
+        </div>
+        <div className="pokemons-compare-data">
+          <div className="dates">
+            <span className="boxes">{pokemonInPokeball[0].height}m</span>
+            <span className="boxes">{pokemonInPokeball[0].weight}kg</span>
+            <span className="boxes">{pokemonInPokeball[0].gender}</span>
+            <span className="boxes">
+              {" "}
+              {pokemonInPokeball[0].abilities.map((abilities) => {
+                return (
+                  <div key={abilities.ability.name}>
+                    {abilities.ability.name}
+                  </div>
+                );
+              })}
+            </span>
+          </div>
```

Comment:
this is repeated bellow, with a different pokemon, an a different class, use a function to avoid repeating it